tests/docker-compose.yml

```diff
@@ -1,6 +1,19 @@
 version: '2.4'
 
 services:
-  app:
+  test:
```

Comment:
Why define a new service instead of overriding app?

Reply:
it didn't occur to me to override. they seem quite different. 

I would say that the intent of `docker-compose -f docker-compose.yml -f tests/docker-compose.yml run test` seems a lot clearer to me than `docker-compose -f docker-compose.yml -f tests/docker-compose.yml run app`

Reply:
It's our current practice to override the application service to run the tests because it reduces repeat configuration code and attempts to ensure as much consistency between app and test environments as possible. (The latter will be more compelling when we start to use our containers in deployed environments, IMO, but it's there nonetheless.) I'd prefer to stick with our pattern and hear proposals for changes over in `how-to`, if that's all right with you?

Reply:
ok